frontend/scripts/vercel.build.sh

```
@@ -0,0 +1,57 @@
+echo "🔹 debug 00"
+# 提取所有 deploy 标签
+# DEPLOY_LABELS=$(echo "$COMMIT_MSG" | grep -oP '\[deploy:\K[^\]]+')
+DEPLOY_LABELS=()
+while [[ "$COMMIT_MSG" =~ \[deploy:([^\]]+)\] ]]; do
+  DEPLOY_LABELS+=("${BASH_REMATCH[1]}")
+  COMMIT_MSG=${COMMIT_MSG#*${BASH_REMATCH[0]}}
+done
+echo "🔹 debug 0"
+DEPLOY_APPS=()
+echo "🔹 debug 1"
+for label in $DEPLOY_LABELS; do
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)